.github/ISSUE_TEMPLATE/task_for_userstory.md

```diff
@@ -0,0 +1,33 @@
+---
+name: Task format for User Story (Japanese)
+about: Create a task for User Story to help us improve
```

Comment:
I think the phrase "to help us improve" is not necessary.  This phrase is a cliché for getting contributions (from non-project members).

Reply:
I deleted the phrase "to help us improve" by review comment.